src/systems/filecoin_mining/sector/sectorset.go

```diff
@@ -19,3 +19,8 @@ func (css *CompactSectorSet) Remove(sectorNo SectorNumber) CompactSectorSet {
 	var newCompactSectorSet CompactSectorSet
 	return newCompactSectorSet
 }
+
+func (css *CompactSectorSet) DeepCopy() CompactSectorSet {
+	var newCompactSectorSet CompactSectorSet
+	return newCompactSectorSet
```

Comment:
does this need a todo?

Reply:
All CompactSectorSet operations are TODOs for now, adding to all